### ISSUE TITLE
Thought process behind the algorithm

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework3G3#22]
I just wanted to clarify that my assumption is that there is a spring attached to the ceiling. Then, a mass is attached at the bottom of the spring. The difference in position before and after the mass is attached is what I define as "x". Then I can assume the force of gravity and the spring force are equal, making it possible to calculate the spring force (given the mass and x).

**Comment by s4il3sh** [comment]
Here is the actual data of the experiment you can use to determine the spring constant.
![image](https://github.com/ubsuny/23-Homework3G3/assets/144289804/8d4c1d0e-8fc4-4cdb-8d78-d81d9667c7b5)

**Comment by WildJimmy** [comment]
@JustinJaniszewski yep, that was my assumption at least. Given that this gives you a general formula to find the spring constant with these parameters
